lib/primer/rails_forms/caption.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if @context.caption? %>
```

Comment:
Let's raise an error if there are both a caption passed in the context _and_ a caption template.